### PR TITLE
Initial design composition styling

### PR DESCRIPTION
**Pull request by jfrankl**
This begins the process of adding styles from the design comp to the application.

image

image

### CODE REVIEW

**Comment by kdeloach** [comment]
@azavea-bot rebuild

File: src/nyc_trees/apps/core/templates/base.html [hunk 1]
```diff
@@ -26,18 +30,26 @@
                     {% endif %}
                     <li><a href="{% url 'home_page' %}">Dashboard</a></li>
                     <li><a href="{% url 'training_summary_pure' %}">Training</a></li>
-                    <li><a href="{% url 'group_list_page' %}">Groups</a></li>
+                    <li class="active"><a href="{% url 'group_list_page' %}">Groups</a></li>
```
Comment (kdeloach) [inline src/nyc_trees/apps/core/templates/base.html]: Is groups supposed to remain active?

Comment (jfrankl) [inline src/nyc_trees/apps/core/templates/base.html]: No, only the active page should be active. What's a good way to handle that
kind of situation, where I want the developers to know that I've made a
class like that, but the functionality doesn't exist to make it work?
Should I just write a todo or a note?

On Mon, Feb 2, 2015 at 10:34 AM, Kevin DeLoach <notifications@github.com>
wrote:

> In src/nyc_trees/apps/core/templates/base.html
> <https://github.com/azavea/nyc-trees/pull/417#discussion_r23931678>:
>
> > @@ -26,18 +30,26 @@
> >                      {% endif %}
> >                      <li><a href="{% url 'home_page' %}">Dashboard</a></li>
> >                      <li><a href="{% url 'training_summary_pure' %}">Training</a></li>
> > -                    <li><a href="{% url 'group_list_page' %}">Groups</a></li>
> > +                    <li class="active"><a href="{% url 'group_list_page' %}">Groups</a></li>
>
> Is groups supposed to remain active?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/azavea/nyc-trees/pull/417/files#r23931678>.
>



-- 
Jeff Frankl, UI/UX Designer

Azavea  |  340 N 12th St, Suite 402, Philadelphia, PA
Web azavea.com  |  Blog azavea.com/blogs  |  Twitter @azavea
<http://twitter.com>

Comment (kdeloach) [inline src/nyc_trees/apps/core/templates/base.html]: I would create an issue for this.

**Comment by kdeloach** [comment]
Nice +1

**Comment by jfrankl** [comment]
Opened issue #418.

**Pull request merged**